content/collections/experiment-theory/en/experiment-set-mde.md

```diff
@@ -12,9 +12,9 @@ updated_at: 1715102912
 ---
 Before you run a experiment, set a Minimum Detectable Effect (MDE) to estimate how you'll measure success. Think of MDE as the minimum change you're hoping to find by running your experiment. Without a fail-safe calculation available for the MDE, it can be tricky to set one. With Amplitude Experiment, the default MDE is 2%. However, as the MDE is directly linked to your unique business needs, be thoughtful during each experiment's [design phase](/docs/feature-experiment/workflow/define-goals). Considerations for setting the MDE should include the primary metric and any associated risks.
 
-## MDE and the experiment type
+## MDE and the metric goal type
 
-When you create your experiment, you select between two experiment types: A/B test or Multi-Armed Bandit.
+When you create your experiment, you select between two metric goal types: success or gaurdrail.
```

Comment:
this bug was probably there because we had success and guardrail on the experiment type like 1 year or 2 ago and we never updated the doc